examples/oft-solana/tasks/solana/retryMessage.ts

```diff
@@ -68,12 +68,9 @@ task('lz:oft:solana:retry-payload', 'Retry a stored payload on Solana')
                     nonce: nonce.toString(),
                     srcEid,
                     sender: makeBytes32(sender),
-                    dstEid,
```

Comment:
context on why dstEid is also deleted

the type for lzReceive's third param:

```
packet: Pick<Packet, "message" | "sender" | "srcEid" | "guid" | "receiver" | "nonce">,
```

dstEid is not needed